packages/firestore/CHANGELOG.md

```diff
@@ -1,8 +1,15 @@
 # Unreleased
+- [changed] Improved iOS 13 support by addressing a change in Mobile Safari
```

Comment:
It's probably more important to tie this to observed user behavior and link to the details if someone is really curious.

> Fixed a crash on iOS 13 that could happen if Firestore was used in a tab in the background (#2232).

Reply:
Updated to say:

"Fixed a crash on iOS 13 that occurred when persistence was enabled in a background tab."

This still points out that it is a persistence-only issue. As for linking to the original issue, we unfortunately don't do this in our public release notes (see https://firebase.google.com/support/release-notes/js), but I added it here in case someone uses our internal release notes.

Reply:
FWIW, we do link to issues in the iOS release notes (https://firebase.google.com/support/release-notes/ios) so there's definitely precedent for it. I think it would be helpful.